### DISCUSSION TITLE
Status=8

### DISCUSSION THREAD

**Discussion by trlafleur** [arendst/Tasmota#14784] [Q&A] [answered]
Every now and then I get an MQTT Status=8, where are these status messages defined...

Thanks...

**Comment by sfromis** [comment, marked as answer]
What do you mean by `Status=8`? Better post the context.

**Comment by trlafleur** [comment]
[image: Screen Shot 2022-02-09 at 8.49.20 AM.png]


when I look at MQTT stream with MQTT Explorer, I see these messages....


~~ _/) ~~~~ _/) ~~~~ _/) ~~~~ _/) ~~

Tom Lafleur

**Reply by trlafleur** [reply]
cmnd/MAINE/TASMOTA//STATUS
cmnd/MAINE/TASMOTA/110180  

with a value of 8

**Reply by sfromis** [reply]
Still not at all clear, but MQTT explorer is not that good a place to look anyway. Better check what is in the Tasmota console at that time.

**Reply by barbudor** [reply]
It means that you have something in your network asking device MAIN/TASMOTA for its sensor status sending the command `STATUS 8`
It's not coming from the device, it's a command sent to the device
In response, the device should answer on `state/MAINE/TASMOTA/RESULT` with value `{ "StatusSNS": .....]` providing a message similar to the SENSOR telemetry message
MQTT explorer has some advantage to display MQTT mesage in a hierachical view
But you don't see the linear view. This could be achieved with a logger or [mosquitto_sub](http://www.steves-internet-guide.com/mosquitto_pub-sub-clients/)

**Comment by trlafleur** [comment]
thanks

~~ _/) ~~~~ _/) ~~~~ _/) ~~~~ _/) ~~

Tom Lafleur
